tests/Operation/UpdateSearchIndexTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Exception\InvalidArgumentException;
+use MongoDB\Operation\UpdateSearchIndex;
+
+class UpdateSearchIndexTest extends TestCase
+{
+    public function testConstructorRequiresANotEmptyIndexName(): void
```

Comment:
If you end up taking the suggestion for DropSearchIndexTest, this could be changed to `testConstructorIndexNameMustNotBeEmpty()`.